apps/rotm/behaviours/back-link-check-report.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  getBackLink(req, res, callback) {
+    let backLink = super.getBackLink(req, res, callback);
```

Comment:
`super.getBackLink` is asynchronous, the implementation should not return a value, but do stuff and then calls back with an error or `null` after an undetermined period of time.

Does `super.getBackLink` need to be async? If so, then you need to change the implementation of `getBackLink` and amend the tests

Reply:
Have you tried running the app with this new behaviour?

Reply:
so is that the case for all async functionality? i.e. they shouldn't return a value?

I just want `getBackLink` to return a value so I am guessing it shouldn't be async